diff --git a/cjs/convert-schema-enum-to-declaration-block-string.js b/cjs/convert-schema-enum-to-declaration-block-string.js
index 4db57412612e478faae6e566a7050a519d6dc1a5..32f5cafec7e5c7a672df94051b3784d596f14e40 100644
--- a/cjs/convert-schema-enum-to-declaration-block-string.js
+++ b/cjs/convert-schema-enum-to-declaration-block-string.js
@@ -77,8 +77,9 @@ const convertSchemaEnumToDeclarationBlockString = ({ schema, node, enumName, enu
             .map(enumOption => {
             const optionName = makeValidEnumIdentifier((0, naming_js_1.convertName)({
                 options: {
+                    useTypesPrefix: false,
                     typesPrefix: naming.options.typesPrefix,
-                    typesSuffix: naming.options.typesPrefix,
+                    typesSuffix: naming.options.typesSuffix,
                 },
                 convert: () => naming.convert(enumOption, {
                     transformUnderscore: true,
diff --git a/esm/convert-schema-enum-to-declaration-block-string.js b/esm/convert-schema-enum-to-declaration-block-string.js
index 4d7438f681220e14cca71741f0875934fd70c4a6..5dfded02990584e45c8fb148adfb85a4f89c4bf8 100644
--- a/esm/convert-schema-enum-to-declaration-block-string.js
+++ b/esm/convert-schema-enum-to-declaration-block-string.js
@@ -74,8 +74,9 @@ export const convertSchemaEnumToDeclarationBlockString = ({ schema, node, enumNa
             .map(enumOption => {
             const optionName = makeValidEnumIdentifier(convertName({
                 options: {
+                    useTypesPrefix: false,
                     typesPrefix: naming.options.typesPrefix,
-                    typesSuffix: naming.options.typesPrefix,
+                    typesSuffix: naming.options.typesSuffix,
                 },
                 convert: () => naming.convert(enumOption, {
                     transformUnderscore: true,
